server/src/main/java/org/opensearch/ratelimitting/admissioncontrol/AdmissionControlSettings.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.ratelimitting.admissioncontrol;
+
+import org.opensearch.common.settings.ClusterSettings;
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.ratelimitting.admissioncontrol.enums.AdmissionControlMode;
+
+/**
+ * Settings related to admission control.
+ * @opensearch.internal
+ */
+public final class AdmissionControlSettings {
+
+    /**
+     * Default parameters for the AdmissionControlSettings
+     */
+    public static class Defaults {
+        public static final String MODE = "disabled";
+    }
+
+    public static final String CPU_BASED_ADMISSION_CONTROLLER = "global_cpu_usage";
```

Comment:
We can move individual controller settings to respective files?

Reply:
Updated.